Documentation/contributor-guide/prow_jobs.md

```diff
@@ -0,0 +1,108 @@
+- [EKS Dashboards](https://monitoring-eks.prow.k8s.io/d/96Q8oOOZk/builds?orgId=1&refresh=30s&var-org=etcd-io&var-repo=etcd&var-build=All&from=now-7d&to=now)
+
+### Panel: “Running / Pending Builds”
+
+Shows the number of builds that are in Running vs Pending states over time.
+Use it to track build backlog or concurrency — e.g., if the “Pending” line rises, builds may be waiting for resources.
+If the “Running” line fluctuates a lot or remains at some steady value, you can infer how many builds typically run in parallel.
+
+### Panel: “Memory Usage per Build”
+
+Shows memory usage over time for each build ID (each build listed in the legend at the bottom).
+The y‑axis shows memory use (e.g., in MiB / GiB).
+Use this to spot builds with unusually high memory usage — a spike indicates one build consumed many resources.
+
+### Panel: “CPU Usage per Build”
+
+Similar to the memory panel but shows CPU usage per build over time. Spikes in CPU usage may indicate heavy compute jobs, inefficiencies, or need for resource tuning.
+
+### Panel: "Resources"
+
+- Memory panel
+
+Green line (“used”): how much memory this build’s pod was using at each time point. Orange/Yellow line (“requested”): how much memory was requested (i.e., Kubernetes requests.memory) for that pod.
+Red line (“limit”): how much memory was limited (i.e., Kubernetes limits.memory) for that pod.
+Y‑axis: shows memory (GiB, MiB) over the build runtime.
+
+X‑axis: time of day/date.
+If the green “used” line is close to or hits the red “limit”, it means the build came close to its memory cap (risking OOM). If “used” is much lower than “requested”, you may be over‑allocating memory (waste).
+If the “requested” line is much higher than “used”, it suggests the job’s request could be tuned downward.
+
+- CPU panel
+
+Similar structure: green = actual usage, orange/yellow = requested CPU, red = CPU limit (if set).
+Y‑axis often in number of CPU cores or fraction thereof (e.g., 1.0 = one core).
+A green line with spikes may show bursts of CPU usage (e.g., build or compile phases) while idle periods show low usage.
+If CPU usage consistently saturates the limit, the job may be throttled or delayed. If usage is consistently far below request, tuning may reduce cost.
+
+It is useful for a few reasons:
+
+1. Tuning resources: By drilling into each build-run, you can determine realistic memory & CPU requests and limits for that job‑type. This helps avoid waste or avoid failed builds hitting resource limits.
+
+2. Spotting anomalies: If one build suddenly used 8 GiB while normally this job uses 1 GiB, it may indicate a regression or mis‑configuration.
+
+3. Capacity planning: Seeing typical and peak usage helps cluster operators plan node sizes, scheduling, concurrency of builds, etc.
+
+4. Debugging performance issues: A build with unexpectedly high CPU or memory might be stuck, looping, or consuming resources inefficiently.
```

Comment:
Move all this to the top of the section; Navigating Performance should start with why it's useful, and then move on to the individual panels, rather than the current flow.